project/Scope.rst

```diff
@@ -0,0 +1,31 @@
+
+Halium's scope primarily consists of 2 parts:
+
+1. Halium's Android is a minimal Android distribution designed to run within an LXC container. It will provide a set of interfaces to let the host of the LXC container utilize Android's HAL to communicate with the hardware of Android-based smartphones. These interfaces come in 3 forms:
+
+    1. Native Android interfaces. (e.g. rild's socket, Audio HAL (dlopen'ed directly), (original) WiFi HAL (`a standard libhybris wrapper <https://github.com/libhybris/libhybris/tree/master/hybris/wifi>`_))
+    2. Libhybris compat layer. Usually used to wrap Android C++ interfaces, contains some logic, and has its counter-part compiled in the Android tree. (e.g. `HWC2 compat layer <https://github.com/libhybris/libhybris/tree/master/compat/hwc2>`_)
+    3. Custom interfaces. This will contain more than a compat layer and may have quite deep integration within Android code. (e.g. Ubuntu Touch's camera interfaces, which consists of `camera compat layer <https://github.com/libhybris/libhybris/tree/master/compat/camera>`_, `custom media compat layer <https://github.com/libhybris/libhybris/tree/master/compat/media>`_, and `deep modifications <https://github.com/ubports/android_frameworks_av/compare/1516fdb...c66dd36>`_ of Android's libmedia). If possible, we would like to reduce this kind of interface as much as possible to reduce our workload, especially those doing deep modification to Android.
+
+2. Halium's middleware is a set of software/libraries/modules on GNU/Linux which talks to the interfaces in 1. and is mostly distro- and toolkit-agnostic. For example, this includes pulseaudio-module-droid, but will not include qtubuntu-camera (a client of the aforementioned camera interface, because it's tied to Qt). Halium might not be upstream of most of the middlewares. Thus, most of the time we'll just maintain the minimal patch(es) that make them work in Halium-based environment.
+
+Halium will primarily be a source-only distribution. Downstream distributions (such as Plasma Mobile or Ubuntu Touch) can place some modification into the Android part (for example, Ubuntu Touch's platform-api, additional kernel cmdline to enable AppArmor, or another initramfs). Then, they can build & distribute Halium as part of the distribution.
+
+Halium will provide the reference rootfs to demonstrate how to integrate Halium into distributions and helps porters in developing their port. It'll contain the common middlewares and enough testing tools to help determine if the port is likely to work in more full-blown distributions or not. A reference repo, which is primarily used to build the reference rootfs, will also be provided. Downstream distributions can use this repo but are not required to.
+
+What is not the scope of Halium
+-------------------------------
+
+Halium does not consist of the following:
+
+  - Deployment tools
+  - Initramfs tooling
+  - Scripts used for the boot sequence (systemd units, init.rc scripts etc.)
```

Comment:
Please make it slightly clear that this is about Linux userspace and not the Android userspace.